install_config/persistent_storage/storage_classes.adoc

```diff
@@ -119,20 +122,21 @@ from plug-in to plug-in.
 [[storage-class-annotations]]
 === StorageClass Annotations
```

Comment:
I might be missing it, but I'm sure if it's clear where this annotation has to be set. This might apply to a few other of these.